src/utils/postErrorToSlack.js

```diff
@@ -0,0 +1,126 @@
+import Auth from '@aws-amplify/auth';
+import { getLoggerBotToken } from 'utils/crypt';
+
+const extractExperimentId = (url) => {
+  const match = url.match(/experiments\/([^/]+)/i);
+  return match ? match[1] : null;
+};
+
+const takeNumLines = (string, numLines) => string.split('\n').slice(0, numLines).join('\n');
+
+// Truncates data arrays so that it doesn't produce large logs
+const ARRAY_TRUNCATE_LENGTH = 21;
+const truncateCollection = (arr) => {
+  if (arr.length < ARRAY_TRUNCATE_LENGTH) return arr;
```

Comment:
what is this exactly truncating? the error message to make it collapsible?

Reply:
No, this truncates large arrays and sets such as cellIds or work results data.